lib/util/getLocalIP.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
```

Comment:
Please use https://www.npmjs.com/package/internal-ip (first choice) or https://www.npmjs.com/package/dev-ip to replace the need for this file.